RELEASE.md

```diff
@@ -0,0 +1,56 @@
+# Releasing the MongoDB Database Tools
+
+This document describes the version policy and release process for the MongoDB Database Tools.
+
+## Versioning
+
+The MongoDB Database Tools use [Semantic Versioning](https://semver.org/).
+
+We will use the following guidelines to determine when each version component will be updated:
+- **major**: backwards-breaking changes to the CLI API or other behaviors (e.g. exit codes) that could break scripts using the Tools
+- **minor**: new features, including new server version support
+- **patch**: bug fixes and cosmetic changes
+
+The team recognizes that log messages, while intended to be informational, may cause breakage if scripts attempt to parse output.
+While we will not commit to a formal versioning policy for log message changes, we will attempt to adhere to the following guidelines:
+- **log.Always** level (the default): adding logging is a "minor" change; changing or removing logging is a "major" change.
+- **log.Info** and higher levels (-v to -vvv): additions, modifications and removals are "patch" changes. We make no promises about message stability at these levels. 
+
+At the moment, there are no pre-release (alpha, beta, rc, etc.) versions of the Tools.
+If there is a need to support pre-release versions of the Tools, we will need to update our release infrastructure to support them.
+
+## Releasing
+
+This section describes the steps for releasing a new version of the Tools.
+
+### Major Release
+
+We do not expect to make any breaking changes to the tools.
```

Comment:
I would expect breaking changes as part of CLI option rationalization at least. Saying the release process doesn't support major releases right now is fine, but I think we should expect to make breaking changes in the coming quarters. I could imagine the index work could result in logging changes for example.

Reply:
agreed

Reply:
updated!